CredentialProvider.Microsoft/Util/CertificateUtil.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Security.Cryptography.X509Certificates;
+using ILogger = NuGetCredentialProvider.Logging.ILogger;
+
+namespace NuGetCredentialProvider.Util
+{
+    internal static class CertificateUtil
+    {
+        public static X509Certificate2 GetCertificateBySubjectName(ILogger logger, string subjectName)
+        {
+            if (string.IsNullOrWhiteSpace(subjectName))
+            {
+                logger.Info(message: Resources.InvalidCertificateInput);
+                return null;
+            }
+
+            var locations = new List<StoreLocation> { StoreLocation.LocalMachine, StoreLocation.CurrentUser };
```

Comment:
nit: use new [] { ... } (or new list init syntax) for stuff like this.. less memory, more terse, and infers the type

Reply:
okay!